manifests/init.pp

```diff
@@ -52,6 +52,7 @@
   $storage_size = $varnish::params::storage_size,
   $varnish_version = $varnish::params::varnish_version,
   $vcl_reload = $varnish::params::vcl_reload,
+  $specific_version = $varnish::params::package_ensure,
```

Comment:
Please correct variable name so it can be referred in `class { 'varnish::install'`

:)